.pep8speaks.yml

```diff
@@ -0,0 +1,2 @@
+pycodestyle:  # Same as scanner.linter value. Other option is flake8
+    max-line-length: 120  # Default is 79 in PEP 8 - sorry IBM 3270 terminal users
```

Comment:
The line with `self._mb = modbus_interface.modbus_interface` was part of the motivation go to up to 120, it would have been a big change to keep it at 79